### PR TITLE
[FIX] correct skip_if logic for text key

### PR DESCRIPTION
**Pull request by sottom**

### CODE REVIEW

**Review by reubano — CHANGES_REQUESTED** [review]
You'll need to rebase and make sure the tests still pass

File: tests/test_examples.py [hunk 1]
```diff
@@ -37,23 +37,17 @@ def test_kazeeki(self):
             'description': (
                 'We are looking for freelancers ( individuals and companies ) '
                 'who offer their services related to Architecture Walkthrough '
-                'and 3D animations. Please consider this job as a potential '
-                'to several more and a long term relationship.   We are a '
-                'Media'),
+                'and 3D animations. Please consider this job as a potential to '
```
Comment (sottom) [inline tests/test_examples.py:40]: I honestly wasn't positive what this test was doing, so I changed the example data to match what it should be.

File: riko/cast.py [hunk 2]
```diff
@@ -157,6 +157,10 @@ def cast(content, _type='text', **kwargs):
     elif kwargs:
         value = CAST_SWITCH[_type]['func'](content, **kwargs)
     else:
-        value = CAST_SWITCH[_type]['func'](content)
+        try:
+            value = CAST_SWITCH[_type]['func'](content)
+        except InvalidOperation:
+            # fix decimal conversion error
+            value = CAST_SWITCH[_type]['func']('NaN')
```
Comment (reubano) [inline riko/cast.py:165]: this commit should be `[FIX]` instead of `[CHANGE]`.

File: tests/test_examples.py (hunk 1 above)
Comment (reubano) [reply tests/test_examples.py:39]: We can discuss today.

**Review by reubano — CHANGES_REQUESTED** [review]

File: tests/test_examples.py (hunk 1 above)
Comment (sottom) [reply tests/test_examples.py:39]: I will take a look at this for an hour to see where the data was going.

**Pull request merged**